src/ape/api/networks.py

```diff
@@ -606,6 +607,18 @@ def config(self) -> PluginConfig:
     def _network_config(self) -> Dict:
         return self.config.get(self.name, {})
 
+    @cached_property
+    def gas_limit(self) -> Union[Literal["auto"], Literal["max"], int]:
+        value = self._network_config.get("gas_limit", "auto")
+
+        if value.isdigit():
```

Comment:
I wonder if we should ever handle hex strs too if that is overkill

Reply:
Added support for it in 4c502af781eca05c73bfa7299393040f58d2038f